duet/views/funnel_ios_new.view.lkml

```diff
@@ -0,0 +1,429 @@
+                      AND first_seen_date >= '2022-01-01'
+                      AND NOT (app_display_version = '107.2' AND submission_date >= '2023-02-01') -- incident filter
+                      GROUP BY 1, 2
+                    ), retention AS (
+                      SELECT client_id, first_seen_date, country
+                           , COALESCE(MAX(IF(submission_date = DATE_ADD(first_run_date, INTERVAL 27 DAY), mozfun.bits28.active_in_range(days_seen_bits, -6, 7), NULL) ), FALSE) AS week_4_retention
+                           , COUNT(DISTINCT CASE WHEN DATE_DIFF(submission_date, first_run_date, day) >= 1 AND DATE_DIFF(submission_date, first_run_date, day) <= 28 AND days_since_seen = 0 THEN submission_date END) AS first_28_days_active_days
+            FROM `mozdata.firefox_ios.baseline_clients_last_seen`
+            WHERE
+              submission_date >= '2022-01-01'
+                      AND first_seen_date >= '2022-01-01'
+              AND NOT (app_display_version = '107.2' AND submission_date >= '2023-02-01') -- incident filter
+            GROUP BY 1, 2, 3
+                    ), ret_aggs as (
+                      SELECT first_seen_date AS date,
+                        country, count(distinct case when first_28_days_active_days > 1 then client_id end) multi_days_users
+                               , count(distinct case when week_4_retention = True then client_id end) week_4_retained
+                      FROM retention
+                      group by 1, 2
+                    )
+                    SELECT date, country, views, downloads, new_profiles, activations, multi_days_users, week_4_retained
+                    FROM top
+                    JOIN bottom USING (date, country)
+                    JOIN ret_aggs USING (date, country)
+;;
+    }
+
+    dimension: country {
+      type: string
+      sql: ${TABLE}.country ;;
+    }
+
+    dimension_group: date {
+      type: time
+      datatype: date
+      timeframes: [date, week, month, year]
+      sql: ${TABLE}.date ;;
+      convert_tz: no
+    }
+
+    measure: views {
+      description: "Unique daily impressions, counted when a customer views the app on the Today, Games, Apps, or Search tabs on the App Store, or on the product page"
+      type: sum
+      sql: ${TABLE}.views ;;
+    }
+
+    measure: downloads {
+      description: "Total downloads, including first-time downloads and redownloads"
+      type: sum
+      sql: ${TABLE}.downloads ;;
+    }
+
+    measure: new_profiles {
+      description: "Unique Client IDs, usually generated when the app is opened for the first time"
+      type: sum
+      sql: ${TABLE}.new_profiles ;;
+    }
+
+    measure: activations {
+      description: "Early indicator for LTV based on days of app open and searches on the first week"
+      type: sum
+      sql: ${TABLE}.activations ;;
+    }
+
+  measure: multi_days_users {
+    description: "Early indicator for LTV based on days of app open in first 28 days"
```

Comment:
Are these metrics meant to be early indicators for LTV? I thought they were just retention metrics.

Reply:
I look at them as some LTV proxy. Especially because they have some associations with long term retention